.gitignore

```diff
@@ -11,6 +11,8 @@ server/manifest.go
 
 assets/i18n/translate.*.json
 
+public/jaas
```

Comment:
Why is this gitignored?

Reply:
make dist will automatically generate folder named Jaas inside the public directory